### PR TITLE
Feature rpf001/002

### PR DESCRIPTION
**Pull request by danielsespitia**
Created Restaurant Profile Form Page.

Once the user is registered as Restaurant, this feature redirects them to a form designed to store basic data about the Restaurant such as name, address, phone, opening and closing hours and a minimum deposit required to schedule a reservation from the client end.

The opening and closing hours and deposit fields values are subject to form validation in order to submit just permitted values.

The Restaurant Profile Form Page contains a form with a submit button that is routed to 'your-profile' page (which doesn't have any rendered data other than its title yet). Also, the profile picture displayed on the form is a place holder. 

Since the data is not being printed yet, console.log is included in the code in order to show the updated form data in the console.

The pages and components created for this feature were:

- [x] RestaurantForm.js
- [x] RestProfileForm.js
- [x] RestaurantProfile.js

1Page
2RequiredHour
3Required Deposit
4PageRouting

### CODE REVIEW

**Review by carosalazar28 — DISMISSED** [review]

File: alamesa/src/App.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ from 'react-router-dom'
 import Header from './components/Header'
 import Home from './pages/Home'
 import FormSignUp from './pages/FormSignUp'
+import RestaurantForm from './pages/RestaurantForm'
+import YourRprofile from './pages/YourRprofile'
```
Comment (simonhoyos) [inline alamesa/src/App.js]: yo le cambiaria el nombre al componente. El nombre es importante para darnos una idea de lo que hace. Un ejemplo `RestaurantProfile`

File: alamesa/src/pages/RestaurantForm.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    phone: '',
+    scheduleFrom: 0,
+    scheduleTo: 0,
+    deposit: 0,
+  };
+
+  handleChange = e => {
+    const { name, value } = e.target;
+
+    this.setState({ [name]: value }, () => console.log('current', this.state))
```
Comment (simonhoyos) [inline alamesa/src/pages/RestaurantForm.js]: Es importante no dejar console.logs en el código que ya está listo (hay algunas excepciones)

File: alamesa/src/pages/YourRprofile.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function YourRprofile() {
```
Comment (simonhoyos) [inline alamesa/src/pages/YourRprofile.js]: Lo mismo de cambiar el nombre

**Review by simonhoyos — CHANGES_REQUESTED** [review]

File: alamesa/src/App.js [hunk 4]
```diff
@@ -17,6 +19,11 @@ function App () {
       <Switch>
         <Route exact path="/" component={Home}/>
         <Route exact path="/sign-up" component={FormSignUp}/>
+        <Route exact path='/' component={Home}/>
```
Comment (andress20) [inline alamesa/src/App.js]: Quedaron 2 route apuntando a Home

**Review by carosalazar28 — DISMISSED** [review]

**Review by andress20 — DISMISSED** [review]

**Review by andress20 — APPROVED** [review]

**Review by simonhoyos — APPROVED** [review]

**Review by SergioJaramilloL — APPROVED** [review]

**Pull request merged**